sdk/dapp-kit/src/themes/themeContract.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { createGlobalThemeContract } from '@vanilla-extract/css';
+
+const themeContractValues = {
+	blurs: {
```

Comment:
I have a WIP that I'm splitting up here (https://github.com/MystenLabs/sui/pull/13994) if you want to see how these definitions are used on the UI